rj_gameplay/rj_gameplay/tactic/line_tactic.py

```diff
@@ -13,7 +12,11 @@ def __init__(self, world_state: stp.rc.WorldState):
 
         # compute move points
         # TODO: make start on side of the field, so this Tactic is actually useful during penalty situations
-        start = (3.0, 0.0)
+        # access the length and the width of the field (minus border), and starts robots there
+        start = (
+            (world_state.field.length_m - world_state.field.border_m) / 2,
```

Comment:
these seem to be the wrong coordinates, do you get the correct behavior on your end when you run sim?